tests/integration/awslambda/test_lambda_integration.py

```diff
@@ -34,8 +39,97 @@
 THIS_FOLDER = os.path.dirname(os.path.realpath(__file__))
 TEST_LAMBDA_PARALLEL_FILE = os.path.join(THIS_FOLDER, "functions", "lambda_parallel.py")
 
+lambda_role = {
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {"Service": "lambda.amazonaws.com"},
+            "Action": "sts:AssumeRole",
+        }
+    ],
+}
+s3_lambda_permission = {
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "sqs:*",
+                "dynamodb:DescribeStream",
+                "dynamodb:GetRecords",
+                "dynamodb:GetShardIterator",
+                "dynamodb:ListStreams",
+                "kinesis:DescribeStream",
+                "kinesis:DescribeStreamSummary",
+                "kinesis:GetRecords",
+                "kinesis:GetShardIterator",
+                "kinesis:ListShards",
+                "kinesis:ListStreams",
+                "kinesis:SubscribeToShard",
+                "logs:CreateLogGroup",
+                "logs:CreateLogStream",
+                "logs:PutLogEvents",
+            ],
+            "Resource": ["*"],
+        }
+    ],
+}
+
+
+class TestSNSSubscription:
```

Comment:
I feel like this belongs to the SNS service more than Lambda since the subscription is managed by the SNS API and the invocation is also purely on the side of SNS.

Reply:
to be more explicit: IMO this shouldn't be in the awslambda test module

@dfangl  your opinion on this?

Reply:
I'm going to go ahead and move this 👍